addons/pause/userscript.js

```diff
@@ -16,6 +16,16 @@ export default async function ({ addon, global, console, msg }) {
   onPauseChanged(setSrc);
 
   while (true) {
+    if (!document.querySelector(".pause-btn")) {
```

Comment:
This adds multiple listeners. Move this up.

Reply:
The listener should go after the button.

Reply:
Is there a reason why this needs to be inside the while loop...?

Reply:
> After it adds the listener, it immediately adds the button. That means that it won't ever add a listener again.

What about going inside then back to the project page?